examples/RM3/wecSimInputFile.m

```diff
@@ -14,20 +14,21 @@
 % waves = waveClass('noWaveCIC');       % Initialize Wave Class and Specify Type  
 
 % Regular Waves  
-waves = waveClass('regular');           % Initialize Wave Class and Specify Type                                 
-waves.height = 2.5;                     % Wave Height [m]
-waves.period = 8;                       % Wave Period [s]
+% waves = waveClass('regular');           % Initialize Wave Class and Specify Type                                 
+% waves.height = 2.5;                     % Wave Height [m]
+% waves.period = 8;                       % Wave Period [s]
```

Comment:
Should we leave this as regular waves for our example case?

Reply:
I'm reverting this input file back to the RM3 example, but this version will remain on Dom's branch on a prior commit.